.github/workflows/build-test-and-analysis.yaml

```diff
@@ -13,6 +13,21 @@ jobs:
     name: Build, test and analysis
     runs-on: ubuntu-latest
 
+    services:
+      db:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: 'postgres'
+          POSTGRES_PASSWORD: '**your-postgres-pass-here**'
```

Comment:
Let's move the POSTGRES_PASSWORD for the production DB in environment variable (it is used for the JWT_SECRET_KEY).
For the test DB we can leave the user & password hardcoded since they are not sensitive data.